content/api-streamer/examples.textile

```diff
@@ -97,25 +97,6 @@ h3. Live Demo:
   </tr>
 </table>
 
-h2(#hackernews). Hacker News - a live stream of the latest tech and startup links submitted on HN
```

Comment:
Why is this deleted OOI?

Reply:
@mattheworiordan the hub product at https://www.ably.io/hub/products/15 no longer exists